aliyun/src/test/java/org/apache/iceberg/aliyun/TestUtility.java

```diff
@@ -65,7 +65,7 @@ public static AliyunOSSExtension initialize() {
     } else {
       LOG.info(
           "Initializing AliyunOSSExtension implementation with default AliyunOSSMockExtension");
-      extension = AliyunOSSMockExtension.builder().silent().build();
```

Comment:
what's the difference without `silent()`?

Reply:
`silent()` was specific to spring-boot to not display the spring-boot banner and extra log.
Now that we are not using spring-boot, but directly implementing the mock using JDK HTTP server, `silent()` is not applicable anymore.